docs/install.md

```diff
@@ -39,53 +39,71 @@ conda install pytorch==1.5.0 torchvision==0.6.0 cudatoolkit=10.1 -c pytorch
 
-`E.g. 1` If you have CUDA 10.1 installed under `/usr/local/cuda` and would like to install
+`E.g.` If you have CUDA 10.1 installed under `/usr/local/cuda` and would like to install
 PyTorch 1.5, you need to install the prebuilt PyTorch with CUDA 10.1.
 
 ```python
 conda install pytorch cudatoolkit=10.1 torchvision -c pytorch
 ```
 
-`E.g. 2` If you have CUDA 9.2 installed under `/usr/local/cuda` and would like to install
-PyTorch 1.3.1., you need to install the prebuilt PyTorch with CUDA 9.2.
+c. Install mmcv, we recommend you to install the pre-build mmcv as below.
 
-```python
-conda install pytorch=1.3.1 cudatoolkit=9.2 torchvision=0.4.2 -c pytorch
+```shell
+pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/{cu_version}/{torch_version}/index.html
+```
+
+Please replace ``{cu_version}`` and ``{torch_version}`` in the url to your desired one. For example, to install the latest ``mmcv-full`` with ``CUDA 11`` and ``PyTorch 1.7.0``, use the following command:
+
+```shell
+pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu110/torch1.7.0/index.html
 ```
 
-If you build PyTorch from source instead of installing the prebuilt package,
-you can use more CUDA versions such as 9.0.
+If it compiles during installation, then please check that the cuda version and pytorch version **exactly"" matches the version in the mmcv-full installation command. For example, pytorch 1.7.0 and 1.7.1 are treated differently.
+See [here](https://github.com/open-mmlab/mmcv#installation) for different versions of MMCV compatible to different PyTorch and CUDA versions.
 
-c. Create a folder called `code` and clone the mmcv repository into it.
+Optionally you can choose to compile mmcv from source by the following command
 
 ```shell
-mkdir code
-cd code
 git clone https://github.com/open-mmlab/mmcv.git
 cd mmcv
+
 git checkout -b v1.2.6 v1.2.6
-pip install -r requirements.txt
-MMCV_WITH_OPS=1 pip install -v -e .
+MMCV_WITH_OPS=1 pip install -e .  # package mmcv-full, which contains cuda ops, will be installed after this step
+# OR pip install -e .  # package mmcv, which contains no cuda ops, will be installed after this step
+cd ..
 ```
 
-d. Clone the mmdetection repository into it. The mmdetection repo is separate from the mmcv repo in `code`.
+Or directly run
+
+```shell
+pip install mmcv-full
+# alternative: pip install mmcv
+```
+
+**Important:** You need to run `pip uninstall mmcv` first if you have mmcv installed. If mmcv and mmcv-full are both installed, there will be `ModuleNotFoundError`.
+
+d. Install [mmdet](https://github.com/open-mmlab/mmdetection.git), we recommend you to install mmdet with pip.
+See [here](https://pypi.org/project/mmdet/2.9.0/) for different versions of mmdet.
+```shell
+pip install mmdet==2.9.0
+```
+
+Optionally you can choose to install mmdetection from [source](https://github.com/open-mmlab/mmdetection.git) by the following command
```

Comment:
Remove L93-100, and give the link to the official installation guide instead. https://github.com/open-mmlab/mmdetection/blob/master/docs/get_started.md

We do not need to repeat them